packages/driver/test/cypress/integration/e2e/focus_blur_spec.js

```diff
@@ -571,8 +605,12 @@ describe('intercept blur methods correctly', () => {
 
     $el.appendTo(cy.$$('body'))
     $el[0].focus()
-    cy.wrap($el[0]).focus()
-    .should('have.focus')
+
+    if (Cypress.browser.family === 'firefox') {
+      cy.wait(0).get('@selectionchange').should('be.called')
```

Comment:
whys this necessary?

all of these conditionals need a comment else its completely unclear and opaque why its being done